en/airframes/airframe_reference.md

```diff
@@ -39,6 +39,34 @@ This page lists all supported airframes and types including
 </tr>
 </tbody></table>
 
+### Dodecarotor cox
```

Comment:
It was recently added in https://github.com/PX4/Firmware/pull/7532.
@imcnanie if you could add a build log of the vehicle to the dev-guide here that would be great!